routes/User.js

```diff
@@ -0,0 +1,109 @@
+const express = require("express");
+const userRouter = express.Router();
+const passport = require("passport");
+const passportConfig = require("../middleware/passport");
+// require('dotenv').config();
```

Comment:
I assume we do need this because we are accessing config variables from process.env (e.g. on line 16)

Reply:
I'm happy that this has been addressed